.tflint.hcl

```diff
@@ -0,0 +1,7 @@
+config {
+  varfile = ["examples/test.tfvars"]
+}
+
+rule "terraform_unused_declarations" {
+  enabled = true
+}
```

Comment:
We should add more rules from here: https://github.com/terraform-linters/tflint/blob/master/docs/rules/README.md

I suggest:

* terraform_comment_syntax
* terraform_deprecated_index
* terraform_deprecated_interpolation	
* terraform_module_pinned_source	
* terraform_module_version	
* terraform_naming_convention
* terraform_standard_module_structure		
* terraform_typed_variables	
* terraform_unused_required_providers	
* 



<br class="Apple-interchange-newline">[terraform_comment_syntax](https://github.com/terraform-linters/tflint/blob/master/docs/rules/terraform_comment_syntax.md)

Reply:
Added, with the exception of terraform module version, rule not found: https://github.com/sparkfabrik/terraform-sparkfabrik-prometheus-stack/runs/5051758323?check_suite_focus=true